### PR TITLE
feat: change to manage versions by JSON file in installation directory

### LINKED ISSUE #45: Manage versions by JSON file in installation directory
## Is your feature request related to a problem? Please describe.

If I change the installation path, the version data will be reset even though I have installed the package.

## Describe the solution you'd like

Make `apm.json` in the installation path and store version data in the file.

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

Now users can keep their data even if they change the installation path.

BREAKING CHANGE: All installation information will be reset.

## Related Issue

Closes #45 

## Motivation and Context

Reference #45 

## How Has This Been Tested?

OS: Windows 10 Home
Version: 21H1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hal-shu-sato** [comment]
@mitosagi Could you please debug this?

**Review by mitosagi — CHANGES_REQUESTED** [review]
I don't see any problem with the code added this time. However, there is a problem with apmJson.

File: src/plugin/plugin.js [hunk 1]
```diff
@@ -445,13 +449,7 @@ module.exports = {
...
     if (filesCount === existCount) {
-      const installedPlugins = store.get('installedVersion.plugin');
-      installedPlugins.push({
-        id: selectedPlugin.id,
-        repo: selectedPlugin.repo,
-        version: selectedPlugin.info.latestVersion,
-      });
-      store.set('installedVersion.plugin', installedPlugins);
+      apmJson.addPlugin(instPath, selectedPlugin);
       this.setPluginsList(instPath);
```
[the remark this answers is not in the document]
Comment (mitosagi) [reply src/plugin/plugin.js:453]: Sorry. This bug seems to be unrelated to PR.

File: src/script/script.js [hunk 2]
```diff
@@ -467,13 +472,7 @@ module.exports = {
...
     if (filesCount === existCount) {
-      const installedScripts = store.get('installedVersion.script');
-      installedScripts.push({
-        id: selectedScript.id,
-        repo: selectedScript.repo,
-        version: selectedScript.info.latestVersion,
-      });
-      store.set('installedVersion.script', installedScripts);
+      apmJson.addScript(instPath, selectedScript);
       this.setScriptsList(instPath);
```
[the remark this answers is not in the document]
Comment (mitosagi) [reply src/script/script.js:476]: Sorry. This bug seems to be unrelated to PR.

**Comment by hal-shu-sato** [comment]
I fixed it.

**Pull request merged**